lib/commands/create.dart

```diff
@@ -127,6 +131,24 @@ class TizenCreateCommand extends CreateCommand {
       if (overwriteMainFile) {
         createdMainFile.copySync(mainFile.path);
       }
+      if (overwritePubspecFile) {
+        final List<String> pubspec = pubspecFile.readAsLinesSync();
+        pubspec.insert(
+          pubspec.indexWhere(
+              (String line) => line.startsWith('dev_dependencies:')),
+          '  # Tizen-specific dependencies.\n'
+          '  messageport_tizen: ^0.1.0\n'
+          '  tizen_app_control: ^0.1.0\n',
+        );
+        pubspecFile.writeAsStringSync(pubspec.join('\n') + '\n');
+
+        await pub.get(
+          context: PubContext.create,
+          directory: projectDir.path,
+          offline: boolArg('offline'),
+          generateSyntheticPackage: false,
+        );
+      }
```

Comment:
This is a temporary solution and I'm preparing another patch that may remove the entire if statement between line 127-153. I'd like to more focus on the template change (in `main.dart.tmpl`) in this PR.